CoinJoin.md

```diff
@@ -0,0 +1,129 @@
+
+An anonymity set is basically the pool of users participating in a specific transaction. The size of an ideal anonymity set in a WabiSabi can vary from 50 to 100 participants in a CoinJoin round. This means your transaction would be mixed with those 49 to 99 other users.
+
+Samourai 
+
+Samourai Wallet uses Whirlpool, a CoinJoin implementation created by its developer team. The service employs a fixed fee structure. Fees range from 5,000 satoshis for the smallest pool (100,000 satoshis) to 1,750,000 satoshis for the largest pool (50,000,000 satoshis). Each satoshi is equal to one hundred millionth of a Bitcoin (0.00000001 BTC), it is the smallest Bitcoin unit.
+
+Whirlpool's anonymity sets vary based on real-time participation within each pool. Since the number fluctuates the user is encouraged to go through multiple mixing rounds to increase their transaction's anonymity. Samourai wallet model aims to provide a balance between costs and privacy.
+
+Bitcoin mixers
+
+There are also options for those wanting a fast and user-friendly Bitcoin mixer. Websites like Whir and Tumber.io let you mix your Bitcoin by entering the Bitcoin address where you want to receive the mixed BTC. After that you need to send the BTC to the address generated by the tool, they will be sent to a BTC pool, where your Bitcoin will be mixed, and after that sent to the address you entered in the first step.
+
+Whir has a 1% fixed coordinator fee.
+Tumber.io fees start at 0.4% and vary according to the amount mixed.
+
+## Could it work for Decred?
+
+As we said before, CoinJoin is a tool exclusive to Bitcoin. Other coin mixers have been developed by other projects, like the Stake Shuffle protocol from Decred.
+
+StakeShuffle is a privacy protocol available at Decred Wallet Decrediton. It is a non-custodial process used to obfuscate ownership of DCR coins. If you use StakeShuffle your outputs are going to turn fully anonymized, and peers of the server won't be able to link outputs and inputs. This technology is based on a paper called " “P2P Mixing and Unlinkable Bitcoin Transactions” by Ruffing, Moreno-Sanchez, and Kate. This system uses a more efficient version known as DiceMix Light, making the process quicker. Although it successfully hides who is sending money to whom, the amounts being sent are still visible to everyone. For added privacy, each mixing transaction uses the same amount of money, known as a fixed denomination.
+
+## Moving Forward
+
+CoinJoin can be highlighted as one of the most used privacy protocols in the crypto market. It is a tool that complements and contrasts with the transparent nature of Bitcoin. For starters learning that Bitcoin is not really anonymous can be a shock, but CoinJoin was created to address this issue.
+
+Have you ever used CoinJoin? Do you think this mixing is better provided by any wallet? Please share your opinion with us in the comments section!
```

Comment:
Generally a very interesting article and well structured.

One minor criticism. When you are writing an article try to ask yourself opposing analytical questions e.g.

If you say something like "This particular wallet is great because .. abc" you should then have an opposing statement to add depth to your argument "but areas it needs improvement are ..xyz".

Building arguments is probably something you are very good at in your current professional life. Try to bring more of those courtroom analytical skills to your writing.

Once you've finished the suggested improvements feel free to build on CT

Reply:
thanks a lot, just finished the suggested improvements, i will pass it to CT